src/blocks/types.tsx

```diff
@@ -470,4 +471,17 @@ export const BlockTypes: { [key: string]: BlockType } = {
     createdBy: "emilee",
     fullscreenEdit: false,
   },
+  "Voice": { 
+    type: "Voice",
+    displayName: "Voice Note",
+    displayDescription: "Use the microphone to record a voice note. The post then allows users to play it back.",
+    emptyTitle: "Empty Voice Note App",
+    emptySubtitle: "Tap here to setup your Voice Note app!",
+    icon: voiceNoteIcon, // TODO: insert your app icon here
+    deprecated: false,
+    doesBlockPost: true,
+    doesBlockEdit: true,
+    createdBy: "Voice Note - yaya sam",
```

Comment:
this'll have to be a seam user name to make sure you get the Seam Points for what you build!

https://seam.mirror.xyz/2lr6chAUva5evedJGFHO5GKuTEvMtaMcNhMehiMfl04

Reply:
My Seam username is @samsam

Reply:
I fixed it in the file